easybuild/framework/easyblock.py

```diff
@@ -56,15 +56,16 @@
 from easybuild.framework.easyconfig.easyconfig import get_easyblock_class, get_module_path, resolve_template
 from easybuild.framework.easyconfig.tools import get_paths_for
 from easybuild.framework.easyconfig.templates import TEMPLATE_NAMES_EASYBLOCK_RUN_STEP
+from easybuild.tools.downloadtools import PROTOCOL_MAP
 from easybuild.tools.build_details import get_build_stats
 from easybuild.tools.build_log import EasyBuildError, print_error, print_msg
 from easybuild.tools.config import build_option, build_path, get_log_filename, get_repository, get_repositorypath
 from easybuild.tools.config import install_path, log_path, read_only_installdir, source_paths
 from easybuild.tools.environment import restore_env
-from easybuild.tools.filetools import DEFAULT_CHECKSUM
-from easybuild.tools.filetools import adjust_permissions, apply_patch, convert_name, download_file, encode_class_name
+from easybuild.tools.filetools import adjust_permissions, apply_patch, convert_name, encode_class_name
+from easybuild.tools.downloadtools import download_file
 from easybuild.tools.filetools import extract_file, mkdir, read_file, rmtree2
-from easybuild.tools.filetools import write_file, compute_checksum, verify_checksum
+from easybuild.tools.filetools import write_file, compute_checksum, verify_checksum, DEFAULT_CHECKSUM
```

Comment:
move the `DEFAULT_CHECKSUM` to the front, and reorder the imported function names alphabetically while you're at it